<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorpusExplorerOptionsDialog</class>
 <widget class="QDialog" name="CorpusExplorerOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Select the corpus item attributes that will be displayed in the Corpus Explorer, as well as the corpus attributes that will be used to group these corpus items. These settings apply to Communications and Speakers.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxCommunications">
     <property name="title">
      <string>Communications</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayoutComDisplay">
        <item row="0" column="0">
         <widget class="QLabel" name="labelComDisplay">
          <property name="text">
           <string>Display in Explorer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTreeView" name="treeViewComDisplay"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayoutComButtons">
        <item row="0" column="0">
         <widget class="QPushButton" name="commandComGroup">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="commandComUngroup">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayoutComGroup">
        <item row="0" column="0">
         <widget class="QLabel" name="labelComGroup">
          <property name="text">
           <string>Group by</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTreeView" name="treeViewComGroup"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="commandComUp">
          <property name="text">
           <string>Up</string>
          </property>
          <property name="icon">
           <iconset resource="../../praaline.qrc">
            <normaloff>:/icons/actions/move_up.png</normaloff>:/icons/actions/move_up.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="commandComDown">
          <property name="text">
           <string>Down</string>
          </property>
          <property name="icon">
           <iconset resource="../../praaline.qrc">
            <normaloff>:/icons/actions/move_down.png</normaloff>:/icons/actions/move_down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxSpeakers">
     <property name="title">
      <string>Speakers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayoutSpkDisplay">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSpkDisplay">
          <property name="text">
           <string>Display in Explorer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTreeView" name="treeViewSpkDisplay"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayoutSpkButtons">
        <item row="0" column="0">
         <widget class="QPushButton" name="commandSpkGroup">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="commandSpkUngroup">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayoutSpkGroup">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSpkGroup">
          <property name="text">
           <string>Group by</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTreeView" name="treeViewSpkGroup"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="commandSpkUp">
          <property name="text">
           <string>Up</string>
          </property>
          <property name="icon">
           <iconset resource="../../praaline.qrc">
            <normaloff>:/icons/actions/move_up.png</normaloff>:/icons/actions/move_up.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="commandSpkDown">
          <property name="text">
           <string>Down</string>
          </property>
          <property name="icon">
           <iconset resource="../../praaline.qrc">
            <normaloff>:/icons/actions/move_down.png</normaloff>:/icons/actions/move_down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../praaline.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CorpusExplorerOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CorpusExplorerOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
